docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/heroku.md

```diff
@@ -257,28 +85,23 @@ export default ({ env }) => ({
 </code-block>
 </code-group>
 
-
-
-You also need to set the `NODE_ENV` variable on Heroku to `production` to ensure this new database configuration file is used.
-
-```bash
-heroku config:set NODE_ENV=production
-```
-
-### 5. Create your Strapi server config for production
-
-Create a new `server.js` in a new [env](/developer-docs/latest/setup-deployment-guides/configurations/optional/environment.md) folder. In this file you only need one key, the `url`, to notify Strapi what our public Heroku domain is. All other settings will automatically be pulled from the default `./config/server.js`.
+4. Create `server.js` inside the `./config/env/production` directory.
+5. Add the code snippet to the `server` configuration file:
 
 <code-group>
 
 <code-block title="JAVASCRIPT">
 
 ```js
-// Path: ./config/env/production/server.js`
+// Path: ./config/env/production/server.js
 
 module.exports = ({ env }) => ({
-  url: env('MY_HEROKU_URL'),
-});
+        proxy: true,
+        url: env('APP_URL'), // replaces `host` and `port` properties in the development environment
```

Comment:
Comment isn't correct, it doesn't replace them persay but sets an internal variable that is used elsewhere. Host and Port physically bind the application to that interface. I think it's safe to simply say something like: "Sets the public URL of your application" and that it's required for heroku deployment